tuf/asn1_codec.py

```diff
@@ -278,13 +278,7 @@ def convert_signed_metadata_to_der(
     # Now sign the metadata. (This signs a cryptographic hash of the metadata.)
```

Comment:
I like how you refer to a specific function when discussing what is returned and given as input (see line 281).   In contrast, the comment that starts on line 278 uses words like `this`, which might lead to confusion.